eel/src/task_manager.rs

```diff
@@ -179,8 +179,12 @@ impl TaskManager {
                                 error!("Connecting to peer {} failed: {}", peer, e);
                             }
                         }
+                        Some(period)
+                    }
+                    Err(e) => {
+                        error!("Failed to query LSP, retrying in 10 seconds: {}", e);
+                        Some(Duration::from_secs(10))
```

Comment:
How did you come up with 10 seconds? Feels pretty long as well.
Also let's put this at least into a constant, if we don't make it configurable (yet).

Reply:
It's what we use for the `reconnect_to_lsp` task. I thought it made sense to be the same, but I agree 10 secs be long. Changed to 5 and created `PeriodConfig` so that it becomes defined together with other periods.